src/components/sections/Filter.js

```diff
@@ -54,13 +77,21 @@ const Filter = () => {
       toggleable
       summary={(
         <p>
-          You can optionally filter which nodes are shown on this stage, by creating
+          You can optionally filter which nodes or edges are shown on this stage, by creating
           one or more rules using the options below.
         </p>
       )}
       startExpanded={!!currentValue}
       handleToggleChange={handleToggleChange}
     >
+      {shouldShowWarning && (
+      <Tip type="warning">
+        <p>
```

Comment:
Just a note to please check that the `Tip` component is typically rendered with a paragraph inside of it, if you haven't already :)

Reply:
Double checked that it is typically rendered with a paragraph inside of it